wiki/People/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/zh.md

```diff
@@ -53,19 +90,23 @@ BN 申请程序于 [NAT 官网](http://bn.mappersguild.com/bnapps)进行。
 | 缩短 (Reduced) | 45-60 天 |
 | 未通过 [BN 考试](/wiki/People/Beatmap_Nominators/Beatmap_Nominator_Test) | 30 天 |
 
-### 在离开后重回 BN 团队
+如果申请者申请失败，但是申请者几乎快达到了通过条件时，冷却期限可能会根据情况缩短。
 
-对于玩家，在离开后重回 BN 团队的要求，与第一次加入的要求不太一样。如果玩家在离开团队之前表现良好，他们在重回团队时的要求可能会适当放宽，甚至可以无要求直接加入。NAT 会提示那些能够快速返回 BN 团队的玩家。
+### 在离开 BN 团队后重新加入
+
+对于离开 BN 团队后重新加入玩家，和第一次加入的玩家的要求不太一样。如果玩家在离开团队之前表现良好，他们在重回团队时的要求可能会适当放宽，甚至可以无要求直接加入。NAT 会提示那些能够快速返回 BN 团队的玩家。
 
 如果你曾是 BN，且你未能通过 BN 申请，则需要满足这些要求，以再次提交申请。
```

Comment:
This line is removed in the latest known version of `en.md`.

Reply:
I forgot to delete the line above